### DISCUSSION TITLE
Getting sponsors/donations

### DISCUSSION THREAD

**Discussion by d8sychain** [jonaswinkler/paperless-ng#321] [Feature Requests]
I understand how maintaining a project takes up time for support, debugging, new features, and balancing that time with personal life, day job, etc. This is a really great project with a good community from what I've seen. Something like [Open Collective](https://opencollective.com/) or [PayPal Me](https://www.paypal.com/paypalme/) might be beneficial to helping Paperless-ng grow.

**Comment by jonaswinkler** [comment]
Paperless is a collaborative effort. There's already one person who has done LOTS of work on the new front end. How would you handle splitting the money fairly between developers?

See also #167.

**Reply by d8sychain** [reply]
I read through https://github.com/jonaswinkler/paperless-ng/issues/167 that you mentioned. Noticed it was fairly recent too. Some pretty good points were made.
> How would you handle splitting the money fairly between developers?

That's a good question and I don't have a good answer.
